### ISSUE TITLE
Reference for external support tickets.

### ISSUE THREAD

**Issue by Maikuolan** [phpMussel/phpMussel#145]
**Reference for external support tickets.**
_For stuff being solved elsewhere; Referencing here for convenience and proper ticketing. This list isn't static; Entries may be added and/or removed without notice. Listed order does not represent priority. List is not comprehensive (issues may exist that aren't listed)._

- [x] (Closed) [False Positive on Image](http://www.spambotsecurity.com/forum/viewtopic.php?f=58&t=4323).
- [x] (Closed) [500 Internal Server Error on Update Page](http://www.spambotsecurity.com/forum/viewtopic.php?f=58&t=4345) *(user successfully solved the problem themselves)*.

**Comment by DanielRuf** [comment]
> False Positive on Image.

Do we have more details about this issue or some other GitHub issue?

**Comment by Maikuolan** [comment]
All the relevant information had been posted to a thread on the phpMussel subforum at Spambot Security, which is unfortunately now unavailable (likely permanently, but we'll see, anyhow). The thread can still be seen (for now, at least) via [Google's cache of the thread](http://webcache.googleusercontent.com/search?q=cache%3Ahttp%3A%2F%2Fwww.spambotsecurity.com%2Fforum%2Fviewtopic.php%3Ff%3D58%26t%3D4323&rlz=1C1NOOH_enAU512AU512&oq=cache%3Ahttp%3A%2F%2Fwww.spambotsecurity.com%2Fforum%2Fviewtopic.php%3Ff%3D58%26t%3D4323&aqs=chrome..69i57j69i58.2327j1j7&sourceid=chrome&ie=UTF-8), but due to that that'll be unavailable too as soon as Google updates their cache of the thread in question, I'll quote the thread here (only two posts):

caxanga334 (Wed Oct 11, 2017 10:29 pm):
> I was uploading a few images to my wordpress and one of then had was blocked by phpMussel. After the block I used maintenance mode to upload the rest of the images.
> ```
> DATE: Wed, 11 Oct 2017 18:33:48 -0300
> IP ADDRESS: xxx.xxx.xxx.xxx
> == SCAN RESULTS / WHY FLAGGED ==
> Command injection attempt detected, 'code"' (dust2_beta_6.png)!
> == MD5 SIGNATURE RECONSTRUCTION (FILE-HASH:FILE-SIZE:FILE-NAME) ==
> b73b7e6a906bb51ef38077880c0a3ab8:2987674:dust2_beta_6.png
> ```
>
> [This is the image that was blocked](https://cdn.gamersalapro.com/i/dust2_beta_6.png).

Maikuolan (Thu Oct 12, 2017 4:19 am):
> Found the problem.. Confirmed false positive (nothing dangerous or wrong with the image itself).
> ![False Positive (dust2_beta_6.png)](https://cdn.pbrd.co/images/GOx8hAn.png)
> The risk of false positives for the general command detection signatures is still kind of high at the moment; Something that'll need to be looked at eventually and probably reworked a little, I think. :-/
>
> Thanks for reporting.

In short: This false positive should only affect users using the "general command detection" signature file, which I would consider as high risk for false positives as is anyhow, and not super high priority, but should be eventually looked at, at some point.

**Comment by Maikuolan** [comment]
https://github.com/phpMussel/Signatures/commit/167e1c242209511b2be6e99bf9cef67d71c780f0 and https://github.com/phpMussel/Signatures/commit/eee54d8c4ed14c6678e3524c07693a49eb5d29a8 adds some warnings and general information about various phpMussel signature files, including the specific signature file responsible for the false positive cited by the first link in the issue's first message.

Given that the forums where the corresponding external support ticket was originally created don't even exist anymore, given that the specific signature file in question isn't particularly important for phpMussel anyway (relative to all the various other available signature files) and has limited benefit (to the extent that I feel it would be easier and simpler to just get rid of it entirely, rather than to completely overhaul the way that phpMussel prepares ANSI-normalised data for use with that type of signature file, which.. would still be good to eventually do anyway, but wouldn't actually assist in closing any currently open issues other than this one, and is very low on the current list of priorities for phpMussel), I feel that the warnings and general information added by the above-mentioned commits are sufficient to regard this particular issue as resolved.

I'm not aware of any more open external support tickets anywhere, so.. I'll close this issue now.
